modules/services/hpos-admin-api.nix

```
@@ -18,8 +18,8 @@ in
 
   config = mkIf cfg.enable {
     systemd.services.hpos-admin-api = {
-      after = [ "network.target" ];
-      wantedBy = [ "multi-user.target" ];
+      after = [ "network.target" "nginx.service" ];
+      wantedBy = [ "multi-user.target" "nginx.service" ];
```

There's no reason `nginx.service` would "want" this to be started. `wantedBy` makes sense only as a guide to when the service should be started (`multi-user.target` effectively means "system is accessible to multiple users == startup finished").

```suggestion
      wantedBy = [ "multi-user.target" ];
```